### PR TITLE
Filled like component with

### LINKED ISSUE #8: Add the like filled svg
Create a component from the attached svg.
[likeFilled.svg.zip](https://github.com/vbetivu-ellation/crunchygram/files/7286243/likeFilled.svg.zip)

### PR DESCRIPTION
**Pull request by SandaB123**

### CODE REVIEW

File: src/components/common/FilledLikeSvg.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+const FilledLikeSvg = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="50px"
+    height="50px"
```
Comment (ileahu-ellation) [inline src/components/common/FilledLikeSvg.js]: set width and height to 24px

**Review by ileahu-ellation — DISMISSED** [review]
🚀

File: src/components/common/FilledLikeSvg.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (vbetivu-ellation) [inline src/components/common/FilledLikeSvg.js]: Let's place this component inside a folder with the same name (FilledLikeSvg). Also, add an index.js file with exports there, to keep the commpon/index.js imports untouched.

Comment (SandaB123) [reply src/components/common/FilledLikeSvg.js]: Done

**Review by vbetivu-ellation — DISMISSED** [review]

**Review by vbetivu-ellation — APPROVED** [review]

**Pull request merged**